tests/snuba/api/endpoints/test_organization_events_measurements_histogram.py

```diff
@@ -0,0 +1,413 @@
+from __future__ import absolute_import
+
+import pytest
+import random
+
+from collections import namedtuple
+from copy import deepcopy
+from datetime import timedelta
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+
+from sentry.utils.samples import load_data
+
+
+HistogramSpec = namedtuple("HistogramSpec", ["start", "end", "measurements"])
+
+
+class OrganizationEventsMeasurementsHistogramEndpointTest(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super(OrganizationEventsMeasurementsHistogramEndpointTest, self).setUp()
+        self.min_ago = iso_format(before_now(minutes=1))
+        self.data = load_data("transaction")
+
+    def populate_measurements(self, specs):
+        start = before_now(minutes=5)
+        for spec in specs:
+            spec = HistogramSpec(*spec)
+            for measurement, count in spec.measurements:
+                for i in range(count):
+                    data = deepcopy(self.data)
+
+                    data["timestamp"] = iso_format(start)
+                    data["start_timestamp"] = iso_format(start - timedelta(seconds=i))
+                    value = random.random() * (spec.end - spec.start) + spec.start
+                    data["transaction"] = "/measurement/{}/value/{}".format(measurement, value)
+
+                    data["measurements"] = {measurement: {"value": value}}
+                    self.store_event(data, self.project.id)
+
+    def as_response_data(self, specs):
+        data = []
+        for spec in specs:
+            spec = HistogramSpec(*spec)
+            for measurement, count in sorted(spec.measurements):
+                data.append(
+                    {"histogram_key": measurement, "histogram_bin": spec.start, "count": count}
```

Comment:
Given that histogram is in the name of this endpoint would response data like `key, bin, count` work?

Reply:
Addressed in 4646cf4